sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/pubsublite/internal/PubsubLiteDlqTest.java

```diff
@@ -471,4 +534,31 @@ public void testPubsubLiteErrorFnWithDedupingSuccess() {
 
     p.run().waitUntilFinish();
   }
+
+  @Test
+  public void testPubSubLiteErrorFnReadProto() {
```

Comment:
I think there should either be an error row that is run through the pipeline to check the error message count, or have a separate failure test case.